synced-files/.github/labels.yml

```diff
@@ -0,0 +1,123 @@
+# These are Exercism-wide labels which are shared across all repos.
```

Comment:
@iHiD I've added comments to help explain what the categories are for. Track maintainers will see this in their generated `.github/labels.yml` file, although they won't see it in their labels of course.

I've also re-ordered the labels and colors to have them progress from light to dark, and replaced the one colors that rendered with black text with the one color I had not yet used but which _did_ render as white text.